augur/curate/titlecase.py

```diff
@@ -43,21 +43,6 @@ def titlecase(text: Union[str, None], articles: Set[str] = {}, abbreviations: Se
     Returns a title cased location name from the given location name
     *tokens*. Ensures that no tokens contained in the *whitelist_tokens* are
     converted to title case.
-
-    >>> articles = {'a', 'and', 'of', 'the', 'le'}
-    >>> abbreviations = {'USA', 'DC'}
-
-    >>> titlecase("the night OF THE LIVING DEAD", articles)
-    'The Night of the Living Dead'
-
-    >>> titlecase("BRAINE-LE-COMTE, FRANCE", articles)
-    'Braine-le-Comte, France'
-
-    >>> titlecase("auvergne-RHÔNE-alpes", articles)
-    'Auvergne-Rhône-Alpes'
-
-    >>> titlecase("washington DC, usa", articles, abbreviations)
-    'Washington DC, USA'
```

Comment:
> Migrate tests from the function comments

Why _move_ these doctests into Cram? I think they are very appropriate as unit tests of this function… (and they also nicely document its expected behaviour).

I can see the value in having similar integration tests in Cram but would think to copy or parallel these rather than move them.

Reply:
Agree, added tests back in https://github.com/nextstrain/augur/pull/1197/commits/c00156ba7d31026aa89fc5ed68ae5215c09b99bd